overlays/haskell.nix

```diff
@@ -172,7 +172,7 @@ final: prev: {
         # Package sets for all stackage snapshots.
         snapshots = import ../snapshots.nix { inherit (final) lib ghc-boot-packages; inherit mkPkgSet stackage excludeBootPackages; };
         # Pick a recent LTS snapshot to be our "default" package set.
-        haskellPackages = snapshots."lts-14.13";
+        haskellPackages = snapshots."lts-15.13";
```

Comment:
I understand the motivation for this, though this should be a separate PR, as it bumps the default compiler in haskell.nix from 8.6 to 8.8.

Reply:
Would you rather have the LTS major version depend on the target architecture in this PR?

Reply:
^^^ Went ahead and added this for now.